include/xtensor/xdynamic_view.hpp

```diff
@@ -92,7 +92,7 @@ namespace xt
         class xfake_slice;
     }
 
-    template <class CT, class S, layout_type L = layout_type::dynamic, class FST = detail::flat_storage_getter<CT>>
+    template <class CT, class S, layout_type L = layout_type::dynamic, class FST = detail::flat_storage_getter<CT, L>>
```

Comment:
should the default value for FST be

`detail::flat_storage_getter<CT, XTENSOR_DEFAULT_TRAVERSAL>` instead.

Reply:
and then we don't need the `order` trait.